--- enblend-enfuse-3.2/src/vigra_impex/Makefile.am.orig	2008-01-25 03:40:53.000000000 +0100
+++ enblend-enfuse-3.2/src/vigra_impex/Makefile.am	2008-09-09 20:07:33.504979766 +0200
@@ -2,4 +2,4 @@
 noinst_LIBRARIES = libvigra_impex.a
 libvigra_impex_a_SOURCES = auto_file.hxx bmp.cxx bmp.hxx byteorder.cxx byteorder.hxx codecmanager.cxx codecmanager.hxx error.hxx gif.cxx gif.hxx hdr.cxx hdr.hxx iccjpeg.c iccjpeg.h imageinfo.cxx jpeg.cxx jpeg.hxx png.cxx png.hxx pnm.cxx pnm.hxx rgbe.c rgbe.h sun.cxx sun.hxx tiff.cxx tiff.hxx viff.cxx viff.hxx void_vector.cxx void_vector.hxx exr.cxx exr.hxx
 #libvigra_impex_a_CXXFLAGS = -I${top_srcdir}/include -O3 -Wall -DHasPNG -DHasJPEG -DHasTIFF
-libvigra_impex_a_CXXFLAGS = -I${top_srcdir}/include -O3 -Wall -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -DNDEBUG -DVIGRA_STATIC_LIB
+libvigra_impex_a_CXXFLAGS = -I${top_srcdir}/include -Wall -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -DNDEBUG -DVIGRA_STATIC_LIB
--- enblend-enfuse-3.2/configure.in.orig	2008-09-07 22:21:47.000000000 +0200
+++ enblend-enfuse-3.2/configure.in	2008-09-09 20:07:21.248980604 +0200
@@ -119,7 +119,7 @@
   CXXFLAGS="$CXXFLAGS -g -DDEBUG -Wall"
   AC_MSG_RESULT(yes)
 else
-  CXXFLAGS="$CXXFLAGS -O3 -ffast-math -DNDEBUG -Wall"
+  CXXFLAGS="$CXXFLAGS -ffast-math -DNDEBUG -Wall"
   AC_MSG_RESULT(no)
 fi
 
